lib/checks/keyboard/landmark-is-top-level-evaluate.js

```diff
@@ -5,17 +5,22 @@ import { getComposedParent } from '../../commons/dom';
 function landmarkIsTopLevelEvaluate(node) {
   var landmarks = getAriaRolesByType('landmark');
   var parent = getComposedParent(node);
+  var nodeRole = node.getAttribute('role') || implicitRole(node);
```

Comment:
If we're going to mess with this, why not use aria.getRole?

Reply:
I wondered that myself but didn't know if there was a specific reason why we didn't before.